packages/app/src/messages/reducers/messages.interface.ts

```diff
@@ -66,6 +66,7 @@ export type MessagesState = Readonly<{
   threadsState: ResultState
   messagesStateMap: { [id: string]: ResultState }
   error: Error | string | null
+  deletingState: ThreadDeletingState | undefined
```

Comment:
To be coherent with the whole project.
```suggestion
  deletingState: ThreadDeletingState | null
```

`undefined` sounds more like an optional variable, `null` meaning that the property was forced as `false value`.

Reply:
Fixed in [c415197](https://github.com/mudita/mudita-center/pull/715/commits/c415197ca7813bd72df51545bd3ab908a237534d)